client/components/settings-group-card/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import FormSectionHeading from 'components/forms/form-section-heading';
+
+const SettingsGroupCard = ( { heading, children } ) => {
```

Comment:
Not sure about a new name. Let's leave it as is, unless @jeffstieler has a good idea before this is ready to merge.